### ISSUE TITLE
Plugin to change className?

### ISSUE THREAD

**Issue by richardgorman** [threepointone/glamor#322]
Hi, I am wondering if it possible to write a plugin that will change the selector that is appended to the elements when they are rendered.

**Comment by threepointone** [comment]
Can you tell me your usecase?

**Comment by richardgorman** [comment]
I just found that I can use a `label` field in my styles but I was hoping to append a new class name to the element.

So instead of `.css-label-hash` I could end up with `.label .css-hash`. Is that currently possible?

**Comment by threepointone** [comment]
Not exactly. You should experiment with the plugin arch, because it does let you rewrite the selector before being written. That said, I'd love to hear more about your actual usecase. Why do you need this?

**Comment by richardgorman** [comment]
What do you mean by plugin arch?

**Comment by threepointone** [comment]
could you tell me your usecase, what you're trying to achieve?

**Comment by richardgorman** [comment]
I am trying to achieve a classname that looks like `.{label} .css-hash` instead of .css-{label}-hash

**Comment by threepointone** [comment]
I get that, am curious to know why and how and why you're using this, and whether there's an alternate solution?
